node/perf-test/src/tests/fibonacci.rs

```diff
@@ -81,7 +81,7 @@ where
 		// do a create() call (which doesn't persist) to see what our expected contract address
 		// will be. afterward we create a txn and produce a block so it will persist.
 		// TODO: better way to calculate new contract address
```

Comment:
Not related to this PR, but @notlesh I think this is what `create2` is for. You provide the salt and then you know the deploy address in advance.